joplin/base/models/guide_page.py

```diff
@@ -53,9 +53,9 @@ class GuidePage(JanisBasePage):
                     ('section_heading_vi', TextBlock(label='Heading [vi]', required=False)),
                     ('pages', ListBlock(PageChooserBlock(label="Page", page_type=[InformationPage, ServicePage]), help_text='Select existing pages in the order you want them to display within each heading. Pages should be added only once to any single guide.')),
                 ],
-                label="Section"
+                label="Section",
             )),
-        ],
+        ], max_num=3),
```

Comment:
what limits the number of blocks, when this is ready for merging I will remove from guide pages